client/engine/messageservice/test-messageservice.go

```diff
@@ -25,6 +25,9 @@ type TestMessageService struct {
 	in       chan protocols.Message // for recieving messages from engine
 	out      chan protocols.Message // for sending message to engine
 	maxDelay time.Duration          // the max delay for messages
+
+	// connection with Peers:
+	fromPeers chan []byte // for receiving serialized messages from peers
```

Comment:
a little nit for your information. In this struct:
- the comment `// connection to Engine:` is interpreted as documentation for `in`
- `// connection with Peers:` as documentation for `fromPeers`
- The actual docstrings intended for each field (eg, `// for receiving serialized messages from peers`) are omitted.

Codebase is full of these, as they are a natural way to document things in js/ts, but we should attempt to habituate ourselves to the godoc style